virtwho/virt/kubevirt/kubevirt.py

```diff
@@ -136,12 +141,12 @@ def getHostGuestMapping(self):
 
         for vm in vms['items']:
             spec = vm['spec']
-            host_name = vm['status']['nodeName']
 
             # a vm is not scheduled on any hosts
-            if host_name is None:
+            if vm['status']['phase'] != 'Running':
```

Comment:
Frankly, virt-who does not really care if the VM is `Running` it only cares if it is mapped to a host. Thus I think that 
```
host_name = vm['status'].get('nodeName')
```
would be a more precise change.

Reply:
As far as I understand how k8s work when a vm is `Running` phase `nodeName` always will be there. Do you suggest to make the changes as minimal as possible?

Reply:
No, my issue is of precision. If for some reason KubeVirt has a bug and does not report the node on a Running VM. "My" suggested code just skips it. If KubeVirt introduces a new status "RunningFast", I don't care. virt-who needs to report mapping of VMs to nodes; it should not care about anything else.

Reply:
OK, I see your point. Will update this PR. Thanks!